lib/node_modules/@stdlib/number/float64/to-json/README.md

```diff
@@ -0,0 +1,99 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2018 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# number2json
+
+> Return a `JSON` representation of a `number`.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var number2json = require( '@stdlib/number/float64/to-json' );
+```
+
+#### number2json( x )
+
+Returns a `JSON` representation of a `number`.
```

Comment:
I was unable to add hyperlink in `JSON`, ```es-lint``` was throwing some error

Reply:
Can you paste the error here?

Reply:
```
Linting file: lib/node_modules/@stdlib/number/float64/to-json/README.md
lib/node_modules/@stdlib/number/float64/to-json/README.md
  23:12-23:40  error  Found reference to undefined definition  no-undefined-references  remark-lint
  45:11-45:39  error  Found reference to undefined definition  no-undefined-references  remark-lint

✖ 2 errors
make: *** [/home/pranavchiku/stdlib/tools/make/lib/lint/markdown/Makefile:72: lint-markdown-files] Error 1

Markdown lint errors.
```

Reply:
```Returns a [JSON][http://www.json.org/] representation of a number.``` 
This is how I am adding the hyperlink

Reply:
Ah. Right. This should be `[JSON][json]`. And then in the `<section class="links">` section at the bottom, do

```
[json]: http://www.json.org/
```

In short, simply replicate what we do in [`@stdlib/buffer/to-json`](https://raw.githubusercontent.com/stdlib-js/stdlib/develop/lib/node_modules/%40stdlib/buffer/to-json/README.md).

Reply:
Yeah, I just noticed that, sorry for that